sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/tests/BlobTests.cs

```diff
@@ -65,38 +69,45 @@ public async Task Blob_IfBoundToTextWriter_CreatesBlob()
         {
             // Arrange
             const string expectedContent = "message";
-            QueueClient triggerQueue = CreateQueue(account, TriggerQueueName);
+            QueueClient triggerQueue = CreateQueue(TriggerQueueName);
             await triggerQueue.SendMessageAsync(expectedContent);
 
             // Act
-            await RunTrigger(account, typeof(BindToTextWriterProgram));
+            await RunTrigger(typeof(BindToTextWriterProgram));
 
             // Assert
-            var container = GetContainerReference(account, ContainerName);
+            var container = GetContainerReference(blobServiceClient, ContainerName);
             Assert.True(await container.ExistsAsync());
             var blob = container.GetBlockBlobClient(BlobName);
             Assert.True(await blob.ExistsAsync());
             string content = await blob.DownloadTextAsync();
             Assert.AreEqual(expectedContent, content);
         }
 
-        private static QueueClient CreateQueue(StorageAccount account, string queueName)
+        private QueueClient CreateQueue(string queueName)
         {
-            var client = account.CreateQueueServiceClient();
-            var queue = client.GetQueueClient(queueName);
+            var queue = queueServiceClient.GetQueueClient(queueName);
             queue.CreateIfNotExists();
             return queue;
         }
 
-        private static BlobContainerClient GetContainerReference(StorageAccount account, string containerName)
+        private static BlobContainerClient GetContainerReference(BlobServiceClient blobServiceClient, string containerName)
         {
-            var client = account.CreateBlobServiceClient();
-            return client.GetBlobContainerClient(ContainerName);
+            return blobServiceClient.GetBlobContainerClient(containerName);
         }
 
-        private static async Task RunTrigger(StorageAccount account, Type programType)
+        private async Task RunTrigger(Type programType)
         {
-            await FunctionalTest.RunTriggerAsync(account, programType);
+            await FunctionalTest.RunTriggerAsync(b => {
+                b.Services.AddAzureClients(builder =>
+                {
+                    builder.ConfigureDefaults(options => options.Transport = AzuriteNUnitFixture.Instance.GetTransport());
+                });
+            }, programType,
+            settings: new Dictionary<string, string>() {
+                // This takes precedence over env variables.
+                { "ConnectionStrings:AzureWebJobsStorage", AzuriteNUnitFixture.Instance.GetAzureAccount().ConnectionString }
```

Comment:
Why did you have to do this?

Reply:
There's `QueueTrigger` used in one of the tests and I had to make queue extension notice the Azurite somehow. Alternatives I tried/considered:
- Open Extension.Queues internals to blob tests so that I can substitute QueueServiceClientProvider from queues (I didn't want that as I'd like to decouple blob extension from queue extension at some point (just use Queues SDK in blobs where necessary).
- Change `QueueTrigger` to `BlobTrigger`. This would be ideal as test would still assert same thing but it made the test suite flaky as it uncovered some temporal coupling with BlobTrigger tests if run on same azurite instance. (I think we'll pursue this direction in the future, but didn't want to creep PR with getting to the bottom of it).
- Use env variable. That would make running whole suite of tests on local box not great experience.